gramps/gen/lib/serialize.py

```diff
@@ -48,6 +48,20 @@ class DataDict(dict):
     object interface.
     """
 
+    def __init__(self, data=None):
+        """
+        Wrap a data dict (raw data) or object
+        with an attribute API. If data is an
+        object, we use it to get the attributes.
+        """
+        if data is None:
+            super().__init__()
+        elif isinstance(data, dict):
+            super().__init__(data)
+        else:
+            super().__init__()
+            self["_object"] = data
```

Comment:
```suggestion
        if isinstance(data, dict):
            super().__init__(data)
        else:
            super().__init__()
            if data:
                self["_object"] = data
```
Possibly simpler formulation?